### DISCUSSION TITLE
How to use i3-gaps on KDE?

### DISCUSSION THREAD

**Discussion by N9C13** [i3/i3#4537] [Q&A] [answered]
Can you tell me how I can merge **KDE** (more specifically the Neon version) + **i3-gaps**? 
If you know some guide on github, or on other sites they would be very comfortable

Thanks

**Comment by orestisfl** [comment, marked as answer]
Never tried it myself but you can check out this
https://github.com/heckelson/i3-and-kde-plasma

**Reply by N9C13** [reply]
Thanks bro
